pasystem/pasystem-tool/tool/src/java/org/sakaiproject/pasystem/tool/handlers/BannersHandler.java

```diff
@@ -89,6 +91,12 @@ protected void handleCreateOrUpdate(HttpServletRequest request, Map<String, Obje
         String uuid = extractId(request);
         BannerForm bannerForm = BannerForm.fromRequest(uuid, request);
 
+        User currentUser = UserDirectoryService.getCurrentUser();
```

Comment:
It would be ideal not to use the cover but if you need to it would be best to init the service in the contstructor like the other services